Project.toml

```diff
@@ -11,15 +11,16 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 
 [compat]
-Distributions = "0.14 - 0.24"
-PDMats = "0.6 - 0.10"
-StatsFuns = "0.6 - 0.9"
-julia = "0.7, 1"
+Distributions = "0.25"
+PDMats = "0.11"
+SpecialFunctions = "2.1"
+StatsFuns = "0.9"
+julia = "1.6"
```

Comment:
It's overdue to remove support for 0.7 - but are there actually any 1.6-specific implementations here? Or would an older Julia version still work? E.g., Distributions still supports 1.3 since it works fine and does not block any features and does not cause any additional maintenance burden.

Reply:
There is no CI workflow, we would have to do that first.